queries/javascript/injections.scm

```diff
@@ -1 +1,6 @@
 ((comment) @jsdoc)
+
+(call_expression
+	function: ((identifier) @_name
+		(#eq? @_name "gql"))
+	arguments: ((template_string) @graphql))
```

Comment:
i did that on the draft PR for graphql, but this would also match for `${thing}` variables injected in the template string right ? so if you have
```javascript
gql`
 ${queries}

  ...some graphql types
`
```
the ${queries} would be highlighted in white.
not sure this is an issue we can resolve right now though

Reply:
![image](https://user-images.githubusercontent.com/24785958/100547916-0ee9ff00-322f-11eb-8f00-50a6bc60c159.png)

It appears to be matching correctly for me. Although `:TSHighlightUnderCursor` seems to be matching the javascript grammar in the template string instead of the graphql grammar. e.g. `directive` is TSString instead of TSKeyword

Reply:
The code of the playground has not be updated for multi-parsers. I just applied some duct tape that it will somehow run with the new API. No idea what it will show you within injected languages.

Reply:
@steelsojka plans an update. He knows better than I how the language-tree is working: https://github.com/nvim-treesitter/playground/issues/11

Reply:
@theHamsta I'm working on updating the API's in nvim-treesitter first before I tackle the playground changes. I have a branch created and plan on getting it completed this weekend.